docs/devel/OCI.md

```diff
@@ -0,0 +1,57 @@
+### Building and pushing a policy to an OCI registry
+
+#### Using policy CLI
+
+The [policy CLI](https://www.openpolicyregistry.io/docs/cli/download) tool can be easily used to build and push a policy to a remote OCI registry using just two simple commands:
+- `policy build <path_to_src> -t <org>/<repo>:<tag>`
+- `policy push <registry>/<org>/<repo>:<tag>`
+A full tutorial is available [here](https://www.openpolicyregistry.io/docs/tutorial)
+
+#### Using OPA and ORAS CLIs
+
+To build and push a policy to a remote OCI registry with the [OPA CLI](https://www.openpolicyagent.org/docs/edge/cli/) and [ORAS CLI](https://oras.land/cli/) you can  use the following commands:
+
+- `opa build <path_to_src>` will allow you to build a bundle tarball from your policy code
```

Comment:
Nit: `will allow you to build a bundle tarball from your policy code`  -> `will allow you to build a bundle tarball from your OPA policy and data files`